sdk/core/azure-core/pom.xml

```diff
@@ -88,6 +88,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
```

Comment:
Is this verified to work with OkHttp too? I know OkHttp supports boringssl but we should verify if the versions are compatible.

Reply:
The versions are compatible,
but with okhttp one more dependency is needed and requires the user to manually setup the Security provider in the code.
Best to handle this in a future PR when okhttp performance becomes important.
and currently okhttp has some issues like resource leaks and exceptions thrown with storage SDK, Need to get it stable before focusing on its perf.
@JonathanGiles

Reply:
That is fine - just file a tech debt issue to track enabling SSL for OkHttp

Reply:
Opened the issue - #6266